packages/compass-e2e-tests/tests/collection-validation-tab.test.ts

```diff
@@ -6,7 +6,7 @@ import { createNumbersCollection } from '../helpers/insert-data';
 
 const NO_PREVIEW_DOCUMENTS = 'No Preview Documents';
 
-describe('Collection validation tab', function () {
+describe.only('Collection validation tab', function () {
```

Comment:
```suggestion
describe('Collection validation tab', function () {
```

Reply:
I'm still debugging e2e tests. Things that touch the ace editor stopped working.